ton_client/src/debot/mod.rs

```diff
@@ -35,13 +35,23 @@ use tokio::sync::Mutex;
 #[derive(Serialize, Deserialize, Default, ApiType, Clone)]
 pub struct DebotHandle(u32);
 
+/// Describes a debot action in a Debot Context.
 #[derive(Serialize, Deserialize, Clone, Debug, ApiType, PartialEq)]
 pub struct DebotAction {
+    /// a short action description. Should be used by Debot Browser as name of
+    /// menu item.
     pub description: String,
+    /// Depends on action type. Can be a debot function name or a print string 
+    /// (for Print Action).
     pub name: String,
+    /// Action type. See structure AcType.

```

Comment:
Don't mention non-api structures. If you want to show action types to client better use `AcType` type instead of `u8` here. Or delete `AcType` mention here

Reply:
Lets delete it